data/card_catalogue/upgradeable-1.json

```diff
@@ -77,29 +77,16 @@
       "image": "res/art/illustration/Lightning Bolt.png"
     }
   },
-  {
-    "title": "Fork Lightning",
-    "upgrade": "Electric Vortex",
-    "tier": "red",
-    "details": {
-      "rules": {
-        "active": "Deal 6 damage and stun to up to 6 visible characters.",
-        "passive": null
-      },
-      "flavour": "\"Ouch\" - Dr. Reed and his 5 assistants.",
-      "image": "res/art/illustration/Fork Lightning.png"
-    }
-  },
```

Comment:
Replaced this with Siphon. Siphon has the same ability, but with a nerfed blackout clause stapled on